Xamarin.Forms.Build.Tasks/VariableDefinitionExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using System.Collections.Generic;
+
+namespace Xamarin.Forms.Build.Tasks
+{
+	static class VariableDefinitionExtensions
+	{
+		public static IEnumerable<Instruction> LoadAs(this VariableDefinition self, TypeReference type, ModuleDefinition module)
+		{
```

Comment:
I like the idea of having this in a separate extension method, but I'd keep the `LdLoc` out of it so it could be used on any object on the stack.

Reply:
I agree that the type conversion can be used without `ldloc`, and I also note the possibility in the first comment of this pull request.

However, I decided to create this convenient method because I saw combinations of the type conversion and `ldloc` so frequently. I think we may have a method dedicated for type conversion and call it from this `LoadAs` method, or something for `Ldfld`, or even directly. I have not made such a generic type conversion method yet because I didn't create something like `Xamarin.Forms.Build.Tasks.FieldExtensions.LoadAs` nor find an appropriate situation to call the type conversion method without knowing an instruction to load an object to the stack.

I can make a change accordingly if you still think the type conversion logic should have a dedicated method in this pull request, so please tell me in that case.